pkg/security/config/config_linux.go

```diff
@@ -0,0 +1,17 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
```

Comment:
nit: you can remove this line, the file suffix does the job

Reply:
Oh, I didn't know, fixed ! Thanks